scripts/cassandra-integration-test.sh

```diff
@@ -52,11 +52,22 @@ apply_schema() {
 run_integration_test() {
   local version=$1
   local schema_version=$2
+  local jaegerVersion=$3
   local cid
-  cid=$(setup_cassandra "${version}")
-  apply_schema "$2"
-  STORAGE=cassandra make storage-integration-test
-  exit_status=$?
+  if [ "${jaegerVersion}" = "v1" ]; then
+    cid=$(setup_cassandra "${version}")
+    apply_schema "$2"
```

Comment:
these two lines are not different between two versions, you can do them outside of the if()